BlogTemplate/Pages/Edit.cshtml.cs

```diff
@@ -15,10 +15,14 @@ namespace BlogTemplate.Pages
     public class EditModel : PageModel
     {
         private BlogDataStore _dataStore;
+        private SlugGenerator _slugGenerator;

```

Comment:
Should mark these fields as `readonly` as a best practice. It ensures that the fields can be set only from the ctor.